client/js/components/assessments/item.jsx

```diff
@@ -7,211 +7,124 @@ import UniversalInput         from "./universal_input";
 export default class Item extends React.Component{
 
   static propTypes = {
-    question         : React.PropTypes.object.isRequired,
-    currentItemIndex : React.PropTypes.number.isRequired,
-    questionCount    : React.PropTypes.number.isRequired,
-    messageIndex     : React.PropTypes.number.isRequired,
-    confidenceLevels : React.PropTypes.bool.isRequired,
-    outcomes         : React.PropTypes.object
+    question                   : React.PropTypes.object.isRequired,
+    currentItemIndex           : React.PropTypes.number.isRequired,
+    questionCount              : React.PropTypes.number.isRequired,
+    messageIndex               : React.PropTypes.number.isRequired,
+    confidenceLevels           : React.PropTypes.bool.isRequired,
+    nextQuestion               : React.PropTypes.func.isRequired,
+    prevQuestion               : React.PropTypes.func.isRequired,
+    confidenceLevel            : React.PropTypes.func.isRequired,
+    submitAssessment           : React.PropTypes.func.isRequired,
+    outcomes                   : React.PropTypes.object,
   };
 
   nextButtonClicked(e){
     e.preventDefault();
-    this.setState({unAnsweredQuestions: null});
-    AssessmentActions.nextQuestion();
-    this.setState({showMessage: false});
+    this.props.nextQuestion();
   }
 
   previousButtonClicked(e){
     e.preventDefault();
-    this.setState({unAnsweredQuestions: null});
-    AssessmentActions.previousQuestion();
-    this.setState({showMessage: false});
-  }
-
-  confidenceLevelClicked(e, currentItemIndex){
```

Comment:
Removed confidence level because we aren't planning to use it. It wouldn't be too hard to add back in later.